qemu/tests/cfg/multi_disk_random_hotplug.cfg

```diff
@@ -31,6 +31,9 @@
                 stg_params = "fmt:virtio,virtio_scsi,usb2"
             usbs += " ehci"
             usb_type_ehci = usb-ehci
+            Host_RHEL.m6:
+                usbs= "ehci"
```

Comment:
@xutian I have checked with the virtual blk / usb feature owner @XueqiangWei / @maxujun , it's ok for the case with this type usb controller on RHEL6 host. Thanks!